.github/pull-request-template.md

```diff
@@ -13,7 +13,19 @@ Changes proposed in this pull request:
 
 **Pre-Merge Checklist**
 
-- [ ] As a PR reviewer, verify code coverage and evaluate if it is acceptable.
+Consider all the following items. If any of them are violated, or you are not sure about them, please add a comment to the PR.
+
+- [ ] Verify code coverage and evaluate if it is acceptable.
+- [ ] Create RN for introduced changes.
+- [ ] If Kubebuilder changes were made, run `make generate-manifests` and commit the changes before merge.
+- [ ] Ensure whether pre-existing managed resources are correctly handled.
+- [ ] Change works on all hyperscalers supported by BTP Kyma Runtime.
+- [ ] There is no upgrade downtime.
+- [ ] For infrastructure changes, check the cost in the hyperscaler.
```

Comment:
Or maybe: If you made infrastructure changes, check if they increase/affect the hyperscaler's costs.